AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md

```diff
@@ -78,7 +82,7 @@ To configure a scene to export an Alembic file, add the [Alembic Exporter compon
 The component can be configured to export the entire scene or individual object hierarchies.
 
 Using the AlembicExporter component automatically disables Draw Call Batching. If the Mesh group is valid after being batched then it will be exported. In some cases the data will be batched multiplied times and the results may change.  If you want to control the Batch settings they can be found in the Rendering section of Player Settings.
-
+> Consider clarifying section; maybe a reminder line about what batching does for people who mark everything as static in their scene for performance and panic when the exporter turns it off and the project starts to slow down
```

Comment:
please give an example of what you would add

Reply:
"Adding an AlembicExporter component to your scene will disable draw call batching in your project. With batching enabled, your scene's geometry merges itself in various configurations every frame, thus somewhat marring the faithfulness of the exporter's geometry recording features.

After you're done recording, you can re-enable both static and dynamic batching for your project in the Rendering section of the Player Settings. Having the exporter run with batching enabled will engender undesirable results such at batched meshes being recorded multiple times per frame and overlapping in your Alembic file in an unsightly fashion."

Reply:
thanks!